javaslang/src-gen/test/java/javaslang/CheckedFunction2Test.java

```diff
@@ -165,15 +165,6 @@ public void shouldLiftTryPartialFunction() {
         assertThat(unknown.getCause().getMessage()).isEqualToIgnoringCase("Unknown MessageDigest not available");
     }
 
-    private static final CheckedFunction2<Integer, Integer, Integer> recurrent1 = (i1, i2) -> i1 <= 0 ? i1 : CheckedFunction2Test.recurrent2.apply(i1 - 1, i2) + 1;
```

Comment:
> doesn't use any volatile variable

No, but it uses an effectively final variable, that's meant to avoid the need for local volatile ... (Shipilev would cry seeing that we still don't understand this).
However, I don't trust `HashMap` in a non synchronized environment either ... maybe I should avoid the double locking and just put everything in a `syncronized` block :/